src/lumigo_tracer/lumigo_utils.py

```diff
@@ -309,13 +309,17 @@ def get_edge_host(region: Optional[str] = None) -> str:
     return host
 
 
-def report_json(region: Optional[str], msgs: List[dict], should_retry: bool = True) -> int:
+def report_json(
+    region: Optional[str], msgs: List[dict], should_retry: bool = True, with_done=True
+) -> int:
```

Comment:
I wouldn't tak it as a parameter.
If we use extension, then we most do it

Reply:
which is more about functionality and less about implementation - a better API design :)